app.json

```diff
@@ -0,0 +1,14 @@
+{
+  "name": "Slackin",
+  "description": "",
```

Comment:
Add a description?
The current repo description might not fit super well: *make your organization / channel public easily*